### DISCUSSION TITLE
{"detail":"The email or password provided is incorrect. Please check for typos and try logging in again."}

### DISCUSSION THREAD

**Discussion by stevenbaert** [open-webui/open-webui#6175] [General]
Running latest version of Open-WebUI on Azure Container, but issue is the same on local Mac containers.

I have only oauth enabled for Google.
When I login via Google I sometimes get the message in title.
Any specifics I have to take into account for containers (login form not enabled, merge account is enabled, but do not seem relevant?)

**Comment by hdemel** [comment]
I'm running into the same issue with authentik SSO and open-webui 0.5.7

Authentik is working with portainer's OAUTH, but open web ui responds with "email or password is incorrect". Seems like something on openweb-ui's end not accepting the OAuth log in.

- OLLAMA_BASE_URLS=http://aivo.domain:7869 #comma separated ollama hosts
      - ENV=prod
      - WEBUI_AUTH=true
      - WEBUI_NAME=AI
      - WEBUI_URL=https://lama.domain.uk
      - ENABLE_OAUTH_SIGNUP=true
      - OAUTH_CLIENT_ID=XXX
      - OAUTH_CLIENT_SECRET=XXX
      - OAUTH_PROVIDER_NAME=SSO
      - OPENID_PROVIDER_URL=https://atnk.domain.uk/application/o/domain-ai-ollama/.well-known/openid-configuration
      - OAUTH_SCOPES=openid email profile
      - OPENID_REDIRECT_URI=https://lama.domain.uk/oauth/oidc/callback
      - OAUTH_MERGE_ACCOUNTS_BY_EMAIL=true
      - ENABLE_LOGIN_FORM=false
      - MAIN_LOG_LEVEL=DEBUG

**Reply by fereshtehnm** [reply]
did you manage to fix this? I'm also stuck...

**Reply by stevenbaert** [reply]
Sorry, can't help you. Didn't had issue anymore.

**Reply by hdemel** [reply]
> did you manage to fix this? I'm also stuck...

Yes I managed to get it working. Turns out you need to keep the “encryption key” option in the provider settings blank. Make sure a “signing key” is selected.

**Reply by fereshtehnm** [reply]
> > did you manage to fix this? I'm also stuck...
> 
> Yes I managed to get it working. Turns out you need to keep the “encryption key” option in the provider settings blank. Make sure a “signing key” is selected.

Oh, thanks. I'll try to find that in Google Auth settings!

**Reply by SchwarzerA** [reply]
Where do I find “encryption key” and “signing key”?

**Comment by ruizcrp** [comment]
I have the same issue - tried everything for over 12h.

**Reply by hdemel** [reply]
Can you post a screenshot of your configuration for the provider? The only issue for me was having the extra key enabled but can take a look and see if it matches what I have.

**Comment by ruizcrp** [comment]
@hdemel thank you for your message, I made it now and it works (with keycloak). Although I don't know why. I think it might not have a link with the openweb-ui options. Things that I tried involved also reducing the versions of keycloak. However, at the very end I did it with the newer one.

**Comment by mobilestack** [comment]
if your google oauth call back url in docker compose file, GOOGLE_REDIRECT_URI, this one, is not correct with your current used url, it will show this error as well.
